domrender/renderer-js-script.js

```diff
@@ -1,7 +1,8 @@
+(function () {
```

Comment:
FYI I'm not a big fan of formatting the JS file because there's no standard and if multiple people do it it can be more hassle than it's worth.  But this once is fine.

Reply:
In fact, there is ;) https://standardjs.com/

And sorry, me IDE didn't use it :(